### PR TITLE
Federated Reporting documentation additions and re-organization

### PR DESCRIPTION
**Pull request by craigcomstock**
Ticket: ENT-6035
Changelog: none

merge together:
https://github.com/cfengine/masterfiles/pull/1828
https://github.com/cfengine/documentation/pull/2361

### CODE REVIEW

File: MPF.md [hunk 1]
```diff
@@ -326,7 +326,7 @@ This [augments file][Augments] will defines `trigger_upgrade` on hosts with IPv4
     "trigger_upgrade": [
       "ipv4_10_10_1",
       "ipv4_10_10_2",
-      "cfengine_3_10_(?!2$)\d+"
+      "cfengine_3_10_(?!2$)\\d+"
```
Comment (nickanderson) [inline MPF.md:329]: Nice catch!

File: MPF.md [hunk 2]
```diff
@@ -402,7 +402,7 @@ This example illustrates enabling management of components on systemd hosts havi
 
 ```json
 {
-  "classes:" {
+  "classes": {
```
Comment (nickanderson) [inline MPF.md:405]: Nice catch.

File: MPF.md [hunk 3]
```diff
@@ -1091,7 +1091,7 @@ For example:
 {
   "vars":{
     "control_common_bundlesequence_end": [ "mybundle1", "mybundle2" ]
-  }
+  },
```
Comment (nickanderson) [inline MPF.md:1094]: good.

File: MPF.md (hunk 1 above)
Comment (craigcomstock) [reply MPF.md:329]: actually, github pointed those out in red! :O

File: MPF.md (hunk 2 above)
Comment (craigcomstock) [reply MPF.md:405]: likewise, credit to github UI

File: MPF.md (hunk 3 above)
Comment (craigcomstock) [reply MPF.md:1094]: again, github showed it in red! :)

**Pull request merged**